petab/visualize/plot_data_and_simulation.py

```diff
@@ -28,14 +34,15 @@ def plot_data_and_simulation(data_file_path: str,
 
     Parameters
     ----------
-    data_file_path: str
-        Path to the data file.
-    condition_file_path: str
-        Path to the condition file.
+    exp_data: str
```

Comment:
```suggestion
    exp_data:
```

As you added typehints (:+1:), it would be good to remove the datatypes from the docstrings. Otherwise we need to update two places. And they are already out of sync ;).

Reply:
yep, you're right